horusmapper.cfg.example

```diff
@@ -200,3 +211,30 @@ bearing_color = black
 
 # Custom bearing color, in hexadecimal #RRGGBB
 bearing_custom_color = #FF0000
+
+
+
+[units]
+
+# unitselection allows choice of metric - the default or imperial1 - horizontal miles and feet for short distances, horizontal miles per hour,  vertical feet, vertical feet per minute
+
+unitselection = metric
+# unitselection = imperial1
+
+
+# Imperial conversion factors
+
+m_to_ft = 3.28084
```

Comment:
I don't think these conversions need to be in the configuration file. They can be hardcoded somewhere. 
Remove the conversions, just leave the unitselection and switch_miles_feet parameters.

Reply:
Removed and left unit_selection and switch_miles_feet